azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-xml-maven.adoc

```diff
@@ -0,0 +1,71 @@
+= Azure Service Bus Management 1.0 XML and Maven Support - Mule 4
+
+Although you can manually code a Mule app in XML, it is more efficient to use Anypoint Studio.
+
+If you manually code a Mule runtime engine (Mule) app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+
+When you install the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location. It also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/<connector>
+http://www.mulesoft.org/schema/mule/<connector>/current/mule-<connector>.xsd
+----
+
+This example shows how the namespace statements are placed in the `<mule>` XML block:
+
+[source,xml,linenums]
+----
+<mule xmlns:<connector>="http://www.mulesoft.org/schema/mule/<connector>"
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/<connector>
+	http://www.mulesoft.org/schema/mule/<connector>/current/mule-<connector>.xsd">
+----
```

Comment:
I would ask the devs to provide the correct name in XML to replace it in  `mule-<connector>` code.